jetty-core/jetty-io/src/main/java/org/eclipse/jetty/io/AbstractRetainableByteBuffer.java

```diff
@@ -75,4 +75,10 @@ public String toString()
     {
         return "%s@%x[rc=%d,%s]".formatted(getClass().getSimpleName(), hashCode(), refCount.get(), BufferUtil.toDetailString(byteBuffer));
     }
+
+    public String toReferenceString()
```

Comment:
Is this really necessary to be exposed publicly?
It is already printed in the `toString()` above.
Actually, I don't think it is necessary.

Reply:
It is not strictly necessary, but if the RBB is being used just as a retainable, then I'd like a way to avoid printing the detail of the buffer twice in the toString above.

Reply:
I solved it a different way.